python-threatexchange/threatexchange/extensions/vpdq/tests/test_vpdq_faiss.py

```diff
@@ -35,6 +38,33 @@
     features = prepare_vpdq_feature(hash, VPDQ_QUALITY_THRESHOLD)
 
 
+def test_utils():
+    assert (
+        get_zero_hash()
+        == "0000000000000000000000000000000000000000000000000000000000000000"
+    )
+    for i in range(10):
+        random_dist = random.randint(0, 255)
+        random_hash = get_random_hash()
+        similar_hash = get_similar_hash(random_hash, random_dist)
+        assert simple_distance(similar_hash, random_hash) == random_dist
+    try:
+        invalid_dist = 256
+        get_similar_hash(get_random_hash(), invalid_dist)
+        assert False
+    except ValueError:
+        assert True
```

Comment:
blocking: use the pytest helper for this.

  https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions